docs/examples/QDevil/QDAC2/GateLeakage.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
looks like the execution of this example during the docs build should be disabled, see here https://qcodes.github.io/Qcodes/examples/writing_drivers/Creating-Instrument-Drivers.html#Documentation